general/lecture1/tasks/_10/Task10.java

```diff
@@ -0,0 +1,43 @@
+package general.lecture1.tasks._10;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/**
+ * Write a program that simulates the guess a number game.
+ * The program should generate a random number between 1 and N (N should be read from the input)
+ * and the user needs to guess it. If the user enters a number less than guessed, the program needs to output “Larger”.
+ * If the user enters a number greater than guessed - the program should output “Smaller”.
+ * If the user guesses the number correctly - the program should output “That’s it!” and finish.
+ **/
+public class Task10 {
+
+    void printSentence(String sentence) {
```

Comment:
It's better to define methods in the appearance order: in a current case, method `guessNumber` uses method `printSentence`. So, `printSentence` should be defined after `guessNumber`